src/Compilers/Core/MSBuildTask/CommandLineBuilderExtension.cs

```diff
@@ -263,5 +263,14 @@ internal void AppendSwitchIfNotNull
                 }
             }
         }
+
+        internal void AppendArgumentsIfNotNull(string arguments)
```

Comment:
>internal void AppendArgumentsIfNotNull(string arguments) [](start = 8, length = 56)

This is just used in one specific caller, so I'd inline.

Reply:
`AppendSpaceIfNotEmpty()`  is a protected method so even if I inline this in the `InteractiveCompiler` I would still have to add method to expose that.

Reply:
I see. OK.

---
In reply to: [285295404](https://github.com/dotnet/roslyn/pull/35734#discussion_r285295404) [](ancestors = 285295404)